### DISCUSSION TITLE
More options around connecting a git repository

### DISCUSSION THREAD

**Discussion by StarfallProjects** [squidfunk/mkdocs-material#7049] [Ideas]
Referring to the feature documented on this page: https://squidfunk.github.io/mkdocs-material/setup/adding-a-git-repository/

Some things it would be nice to be able to do:

1. Enable 'Edit this page' without displaying the repo URL in the header. Currently working round this with some CSS to hide `.md-header__source`, but would be more elegant if done via settings.
2. Allow edit this page to work with a docs repo URL, while displaying the product repo URL in the header (i.e. support for two different sets of links)
3. Allow the 'Edit this page' button to open the page in a new tab.

Not at all urgent/important, just ideas for nice-to-haves.

**Comment by alexvoss** [comment]
All sensible things but I think customization (with CSS and/or template changes) is the right approach for this as there is no way to make everything configurable. Think how big the documentation would be if there was an option for everything! Never mind the amount of code that would need to be maintained.

**Comment by squidfunk** [comment]
All achievable with customization – just override [`actions.html`](https://github.com/squidfunk/mkdocs-material/blob/master/src/templates/partials/actions.html). Please understand that we cannot offer options for everything. We need to keep defaults and the number of settings sensible and maintainable.
